roles/ceilometer-common/tasks/logging.yml

```diff
@@ -9,3 +9,6 @@
       - compress
       - notifempty
       - delaycompress
+
+- name: remove service default log rotate for ceilometer
```

Comment:
no faulty for rhosp provided logrotate conf,  just different from what we provided for ubuntu.
I was trying to keep the conf same for different OS.

/var/log/ceilometer/*.log {
    weekly
    rotate 4
    missingok
    compress
    minsize 100k
    size 10M
    copytruncate
}